app/src/main/res/layout/add_reminder.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:context="com.example.android.navigation.AddReminderFragment">
+
+<androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.appcompat.widget.Toolbar
```

Comment:
A horizontal line appears in the middle of the Label EditText. This might be because the structure of this layout is odd. I'm not sure why you have both a Toolbar and an ActionBar. You should have one or the other and make sure child  views are properly contained in them.

Reply:
Fixed layout issues